lisa/tools/chmod.py

```diff
@@ -12,5 +12,7 @@ def command(self) -> str:
     def chmod(self, path: str, permission: str, sudo: bool = False) -> None:
         self.run(f"{permission} {path}", sudo=sudo, force_run=True)
 
-    def update_folder(self, path: str, permission: str, sudo: bool = False) -> None:
-        self.run(f"-R {permission} {path}", sudo=sudo, force_run=True)
+    def update_folder(
+        self, path: str, permission: str, sudo: bool = False, timeout: int = 600
```

Comment:
set default to 10. It doesn't make sense to call this method for 600 seconds by default. And because the default value is 10, it doesn't need to use it in above called.

Reply:
This argument is added in this change list, the used place set it to 10 seconds. We should use 10 as default value here, unless we found it's not enough. To reduce waiting time, when it hangs, we should try small value firstly.